package.json

```diff
@@ -19,13 +19,14 @@
     "npm": ">=3.0.0"
   },
   "scripts": {
-    "postinstall": "cd lib || builder run npm:postinstall",
```

Comment:
Will this still need the postinstall script? A postinstall script like this will enable this repo to be installable from a git repo.